plopTemplates/reactAdapters/index.stories.tsx.hbs

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import {{pascalCase name}} from './lib';
+import {storiesOf} from '@storybook/react';
+import {{{pascalCase name}}} from './lib';
```

Comment:
I still haven't figured out how to fix this one

Reply:
https://handlebarsjs.com/expressions.html#whitespace-control

So should be this i think

```
{ {{~pascalCase name~}} }
```

Reply:
Thanks! included this in the latest commit